Dockerfile-alpine.template

```diff
@@ -54,6 +54,8 @@ RUN set -eux; \
 	chmod -R ugo=rwX config db sqlite; \
 	find log tmp -type d -exec chmod 1777 '{}' +
 
+# build for musl-libc, not glibc
+ENV BUNDLE_FORCE_RUBY_PLATFORM 1
```

Comment:
Oh man, this is hairy.  I'd suggest we use `export` in the `RUN` line instead of `ENV` but we have to do `bundle install` at runtime too so this makes sense. :disappointed:

I think we should include some links though, maybe something like this?

```suggestion
# build for musl-libc, not glibc (see https://github.com/sparklemotion/nokogiri/issues/2075, https://github.com/rubygems/rubygems/issues/3174)
ENV BUNDLE_FORCE_RUBY_PLATFORM 1
```